src/cryptoadvance/specter/devices/hwi/jade.py

```diff
@@ -671,9 +679,10 @@ def _get_device_entry(device_model: str, device_path: str) -> Dict[str, Any]:
 
         client = None
         with handle_errors(common_err_msgs["enumerate"], d_data):
-            client = JadeClient(device_path, password, expert, chain, timeout=1)
-            d_data["fingerprint"] = client.get_master_fingerprint().hex()
-
+            client = JadeClient(device_path, password, expert, chain, unlock, timeout=1)
+            # We don't get the fingerprint from a locked Jade
+            if client and unlock:
+                d_data["fingerprint"] = client.get_master_fingerprint().hex()
         if client:
             client.close()
 
```

Comment:
Are you intending to PR this stuff up to HWI ?  [ if not, I might ... or something v similar to make Jade more consistent with ledger/trezor when locked ... ]  But happy to leave to you if you were planning to ... ?

Reply:
Happy to leave this up to you :-)